src/Fitting/ArchimedeanFit.jl

```diff
@@ -0,0 +1,363 @@
+    V, vmeta = vcov_method === :jackknife_obs ? _vcov_jackknife_obs(CT, U; estimator=:irho, epsρ=epsρ) :
+             error("vcov_method no soportado")
+    return C, (; estimator=:irho, epsρ=epsρ, vcov=V, vcov_details=vmeta)
+end
+
+# ============================= :IBETA =======================================
+function _fit_ibeta_1par(::Type{CT}, U::AbstractMatrix;
+                         epsβ::Real=1e-10, max_expand::Int=20) where {CT<:ArchimedeanCopula}
+    # data
+    d = (size(U,1) ≥ 2) ? size(U,1) : size(U,2)
+    d ≥ 2 || throw(ArgumentError("fit_ibeta(Archimedean) requires d≥2."))
+    Up = _as_pxn(d, U)
+
+    # β̂ multivariante (Hofert–Mächler–McNeil, ec. (7))
+    β̂ = clamp(blomqvist_beta(Up), -1 + epsβ, 1 - epsβ)
+
+    GT = generatorof(CT)
+    θlo, θhi = θ_bounds(GT, d)
+    a = isfinite(θlo) ? nextfloat(float(θlo)) : -5.0
+    b = isfinite(θhi) ? prevfloat(float(θhi)) :  5.0
+    if !(a < b); a, b = b, a; end  # ensure order
+
+    f(θ) = begin
+        Cθ = ArchimedeanCopula(d, GT(θ))
+        β(Cθ) - β̂
+    end
+
+    fa = f(a); fb = f(b)
+
+    #If there is at least one infinite bound and there is no change of sign, we expand
+    if ( !isfinite(θlo) || !isfinite(θhi) ) && sign(fa) == sign(fb)
+        k = 0
+        while sign(fa) == sign(fb) && k < max_expand
+            if !isfinite(θhi)
+                b *= 2
+                fb = f(b)
+                if sign(fa) != sign(fb); break; end
+            end
+            if !isfinite(θlo) && sign(fa) == sign(fb)
+                a *= 2
+                fa = f(a)
+            end
+            k += 1
+        end
+    end
+
+    # If there is still no bracket, β̂ is outside the achievable range → nearest extreme
+    if sign(fa) == sign(fb)
+        θstar = (abs(fa) <= abs(fb)) ? a : b
+        return ArchimedeanCopula(d, GT(θstar))
+    end
+
+    # Root by Brent (uses Roots.jl; if you've already imported it, you can leave Roots.find_zero)
+    θ = Roots.find_zero(f, (a, b), Roots.Brent(); xatol=1e-10, rtol=0.0)
+
+    if isfinite(θlo) && θ ≤ θlo; θ = nextfloat(float(θlo)); end
+    if isfinite(θhi) && θ ≥ θhi; θ = prevfloat(float(θhi)); end
+
+    return ArchimedeanCopula(d, GT(θ))
+end
+
+
+function _fit_ibeta(::Type{CT}, U::AbstractMatrix;
+                    epsβ::Real=1e-10, vcov_method::Symbol=:jackknife_obs) where {CT<:ArchimedeanCopula}
+    _assert_oneparam(CT)
+    C = _fit_ibeta_1par(CT, U; epsβ=epsβ)
+    V, vmeta = vcov_method === :jackknife_obs ?
+        _vcov_jackknife_obs(CT, U; estimator=:ibeta, epsβ=epsβ) :
+        error("vcov_method no soportado")
+    return C, (; estimator=:ibeta, epsβ=epsβ, vcov=V, vcov_details=vmeta)
+end
+# ============================= :MLE =======================================
+# Seed
+function _seed1(::Type{CT}, d::Integer, Up::AbstractMatrix, start) where {CT<:ArchimedeanCopula}
+    GT = generatorof(CT)
+    if start === :itau
+        return Distributions.params(_fit_itau_1par(CT, Up))[end]
```

Comment:
Here too, you are reinventing dispatch... Simply define 

```julia
_fit(T::Type{Archimedean}, u::Abstractmatrix, ::Val{:default}) = _fit(T, u, ::Val{:mle}) # to set the default. 
_fit(::Type{Archimedean}, ::Abstractmatrix, ::Val{:itau}) = ...  # with the real estimator. 
_fit(::Type{Archimedean}, ::Abstractmatrix, ::Val{:irho}) = 
_fit(::Type{Archimedean}, ::Abstractmatrix, ::Val{:ibeta})
_fit(::Type{Archimedean}, ::Abstractmatrix, ::Val{:mle})
```
and the same for other models, keep one function name and overlad its methods